qiita_db/search.py

```diff
@@ -194,7 +200,8 @@ def __call__(self, searchstr, user):
         Metadata column names and string searches are case-sensitive
         """
         study_sql, sample_sql, meta_headers = \
-            self._parse_study_search_string(searchstr, True)
```

Comment:
Yes, but this is the only way to build a working SQL that will auto-filter the samples. I can go back to doing it pythonically but it is slower.